fuel-p2p/Cargo.toml

```diff
@@ -18,13 +18,20 @@ bincode = "1.3"
 futures = "0.3"
 futures-timer = "3.0"
 ip_network = "0.4"
-libp2p = "0.41"
 rand = "0.8"
 serde = { version = "1.0", features = ["derive"] }
 sha2 = "0.9"
 tokio = { version= "1.17", features = ["sync"] }
 tracing = "0.1"
 
+[dependencies.libp2p]
+version = "0.43"
+features = [
+    "dns-tokio", "gossipsub", "identify", "kad", "mdns", "mplex", "noise", 
+    "ping", "request-response", "secp256k1", "tcp-tokio", "yamux", 
+]
+default-features = false
```

Comment:
Maybe putting this all into a `libp2p = ...` isn't the cleanest, but if the alternative is having to do more work on the toml linter, then just leave it ugly and files some good first issues to follow up.